src/create-job.ts

```diff
@@ -111,15 +114,24 @@ export type JobDescription = {
  * @param description An object that describes the job to create.
  */
 export default async function createJob(description: JobDescription, pipelineData?: any, encodingProfileData?: any, concurrency: boolean = true) {
-  logger.info(`Creating job ${description.name}.`);
+  logger.info(`Creating job ${description.name}`);
 
-  let pipeline: Pipeline | undefined = undefined;
+  let pipeline: any | undefined = undefined;
   if (pipelineData && encodingProfileData) {
     pipeline = await loadPipelineFromObjects(pipelineData, encodingProfileData);
-  } else if(typeof description.pipeline === 'object') {
+  } else if(instanceOfLocalPipelineConfiguration(description.pipeline)) {
     pipeline = new LocalPipeline({...description.pipeline, ...description.encodingProfile as Record<string,string>} );
-  } else {
+  } else if(typeof description.pipeline === 'string'){
     pipeline = await loadPipeline(description.pipeline, description.encodingProfile as string);
+  } else if(instanceOfEncorePipelineConfiguration(description.pipeline)){
+    pipeline = new EncorePipeline(description.pipeline);
+    const configuration: EncorePipelineConfiguration = description.pipeline;
+    const instance: EncoreInstance = await pipeline.createEncoreInstance(configuration.apiAddress, configuration.token, configuration.instanceId, configuration.profile);
+    await delay(configuration.encoreInstancePostCreationDelay_ms); // Delay required to allow instance to be created before calling it
```

Comment:
Feels like this can cause some headache in the future if the instance doesn't exist even after a short delay. Is it possible to maybe check a health check endpoint of the Encore instance to see if it's up and running?